libuavcan/src/transport/uc_can_acceptance_filter_configurator.cpp

```diff
@@ -16,24 +16,22 @@ const unsigned CanAcceptanceFilterConfigurator::DefaultAnonMsgID;
 
 int16_t CanAcceptanceFilterConfigurator::loadInputConfiguration(AnonymousMessages load_mode)
 {
-    multiset_configs_.clear();
```

Comment:
But I delete it  in case the addFilterConfig() was called before computeConfiguration(). Or/and if configs number > available HW  when you call applyConfiguration, than the computeConfiguration called, thus loadInputConfiguration